tests/setup.php

```diff
@@ -3,13 +3,19 @@
 
 require_once dirname(__DIR__) . '/phpdotnet/phd/constants.php';
 require_once __INSTALLDIR__ . DIRECTORY_SEPARATOR . "phpdotnet" . DIRECTORY_SEPARATOR . "phd" . DIRECTORY_SEPARATOR . "Autoloader.php";
+
 Autoloader::setPackageDirs([__INSTALLDIR__]);
 spl_autoload_register(["phpdotnet\\phd\\Autoloader", "autoload"]);
 
-require_once __INSTALLDIR__ . DIRECTORY_SEPARATOR . "phpdotnet" . DIRECTORY_SEPARATOR . "phd" . DIRECTORY_SEPARATOR . "functions.php";
-
 $config = new Config;
 
+$outputHandler = new OutputHandler($config);
+
+$errorHandler = new ErrorHandler($outputHandler);
+$olderrrep = error_reporting();
+error_reporting($olderrrep | VERBOSE_DEFAULT);
+set_error_handler([$errorHandler, "handleError"]);
```

Comment:
Do we require PHP 8.1 or not? As could use:
```suggestion
set_error_handler($errorHandler::handleError(...));
```
if yes